src/Kunstmaan/TranslatorBundle/Command/ExportTranslationsCommand.php

```diff
@@ -3,14 +3,39 @@
 namespace Kunstmaan\TranslatorBundle\Command;
 
 use Kunstmaan\TranslatorBundle\Model\Export\ExportCommand;
+use Kunstmaan\TranslatorBundle\Service\Command\Exporter\ExportCommandHandler;
 use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Serializer\Exception\InvalidArgumentException;
 
+/**
+ * @final since 5.1
+ * NEXT_MAJOR extend from `Command` and remove `$this->getContainer` usages
+ */
 class ExportTranslationsCommand extends ContainerAwareCommand
 {
+    /**
+     * @var ExportCommandHandler
+     */
+    private $exportCommandHandler;
+
+    public function __construct($translationRepository = null)
```

Comment:
Docblock

Reply:
Use same variable name translationRepository <--> exportCommandHandler

Reply:
All docblocks are fixed + correct typehints are added in commands so we can "activate" those in 6.0